### DISCUSSION TITLE
MultiSelectFilter options() not working/showing?

### DISCUSSION THREAD

**Discussion by xavvey** [filamentphp/filament#2968] [General]
This must be something really simple that I am probably overlooking, but following the documentation on filters I am trying to create a MultiSelectFilter for gender on a table. I am only trying to populate the options dynamically.

I have this:
`MultiSelectFilter::make('gender')
                ->options(
                    Profile::query()
                        ->select('gender')
                        ->orderBy('gender')
                        ->get()
                        ->keyBy('gender')
                        ->map(fn($profile) => $profile->gender)
                        ->toArray()
                )
                ->column('gender'),`

The options stay empty..

Using the example from the documentation:
`MultiSelectFilter::make('status')
                ->options([
                    'draft' => 'Draft',
                    'reviewing' => 'Reviewing',
                    'published' => 'Published',
                ])
                ->column('status_id')`

The options also stay empty as shown below:
![image](https://user-images.githubusercontent.com/77445338/176411435-662ac99f-a077-42c1-827b-50847cd297bb.png)

At first I thought it was me doing something wrong while querying for all genders, but seeing as the 'simple' example also doesn't work...

Any idea what I am doing wrong/overlooking?

**Comment by danharrin** [comment]
Are you using the admin panel?

**Reply by xavvey** [reply]
No, I am not. I am using the Table/Form builder in my project and have followed the seperate installation instruction for those.

**Reply by danharrin** [reply]
Are there any console errors? Are you importing the FormsAlpinePlugin? Have you recompiled your assets recently? Have you cleared your view cache?

**Reply by xavvey** [reply]
I had some ReferenceErrors in my console yeah.
I tried recompiling assets and clearing view cache but that didn't work. I think it might had something to do with me using Alpine through cdn instead of npm install? after installing everything through npm those errors had gone away.

still had some issues with babel-runtime-regenerator but installing that, recompiling and clearing view cache seemed to do the trick there

**Reply by danharrin** [reply]
Did you not import the FormsAlpinePlugin then? Because that doesn't work with the CDN, it needs to be part of your build to process the correct theme CSS.

Glad you got it working.

**Reply by xavvey** [reply]
I am not entirely sure to be honest. I thought I had done everything needed for it but could be I missed a step somewhere along the line.
And yeah, I am also glad I got it to work. This is an awesome package so keep up the good work!
